Gemfile.lock

```diff
@@ -368,7 +368,7 @@ GEM
       nokogiri (>= 1.5.10)
     ruby_dep (1.3.1)
     safe_yaml (1.0.4)
-    saml_idp (0.3.1)
+    saml_idp (0.3.2)
```

Comment:
Could we make this update into a separate PR?  Its not clear which parts of this change are due to the new specs vs the the gem update.